hack/lib/util/golang.sh

```diff
@@ -11,7 +11,8 @@ function os::golang::verify_go_version() {
 	if [[ "${go_version[2]}" != go1.8* ]]; then
 		os::log::info "Detected go version: ${go_version[*]}."
 		if [[ -z "${PERMISSIVE_GO:-}" ]]; then
-			os::log::fatal "Please install Go version ${OS_REQUIRED_GO_VERSION} or use PERMISSIVE_GO=y to bypass this check."
+			os::log::warning "Please install Go version ${OS_REQUIRED_GO_VERSION} or use PERMISSIVE_GO=y to bypass this check."
+			return 0
```

Comment:
not return 1?

Reply:
Oh, this is intentional. Hmm. Ok.

Reply:
> not return 1?

No. The bash conditionals isn't right and something doesn't handle detection well.  We hit this last time too.